pinot-controller/src/main/java/org/apache/pinot/controller/validation/RealtimeSegmentValidationManager.java

```diff
@@ -172,8 +183,13 @@ public void cleanUpTask() {
     _validationMetrics.unregisterAllMetrics();
   }
 
+  public static final class RealtimeSegmentValidationTaskParameters {
```

Comment:
This is already an inner class of `RealtimeSegmentValidationManager`, so no need to have a separate wrapper class for the parameters. Suggest directly adding `_recreateDeletedConsumingSegment` into the `Context` class

Reply:
Ack